src/main/java/org/wildfly/security/http/impl/BasicAuthenticationMechanism.java

```diff
@@ -101,7 +100,7 @@ public void evaluateRequest(final HttpServerRequest request) throws HttpAuthenti
         } catch (HttpAuthenticationException e) {
             throw e;
         } catch (IOException e) {
-            throw ElytronMessages.log.mechCallbackHandlerFailedForUnknownReason(BASIC_NAME, e).toHttpAuthenticationException();
+            throw httpBasic.mechCallbackHandlerFailedForUnknownReason(BASIC_NAME, e).toHttpAuthenticationException();
```

Comment:
please, as there are standalone loggers now instead, remove them from message params - most of messages called with "String mechName" as first parameter (you need to chance method signature and remove [%s] from message in ElytronMessages file)

Reply:
(always when there is BASIC_NAME in log message in BasicAuthenticationMechanism and so on...)